fmriprep/utils/static_files.py

```diff
@@ -0,0 +1,373 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import os
```

Comment:
:)

I just copied it all to get these changes put in. Do you think it would be best to have mriqc as a dependency and import it from there? Or to seperate the utils out into its own repo and have mriqc and preproc import it from there? Having the code duplicated is not ideal.